middleware/authorization.js

```diff
@@ -56,32 +57,40 @@ function generateRefreshToken(rowInfo) {
 async function reIssueToken(req, res, next) {
     const accessToken = verifyToken(req.headers["authorization"], ACCESS_PRIVATE_KEY);
     const refreshToken = verifyToken(req.headers["refresh"], REFRESH_PRIVATE_KEY);
-
-    if (accessToken === "TokenExpiredError" && refreshToken === "TokenExpiredError") {
+    console.log("accessToken", accessToken);
+    console.log("refreshToken", refreshToken);
+    if ((accessToken === "TokenExpiredError") && (refreshToken === "TokenExpiredError")) {
 
         return res.redirect("/auth/login");
     }
 
-    if (accessToken === "TokenExpiredError" && refreshToken) {
+    if ((accessToken === "TokenExpiredError") && refreshToken) {
```

Comment:
```suggestion
    if (accessToken === "TokenExpiredError") {
```

기존의 조건문은 `refresh`헤더를 설정하지 않았을때 `verifyToken`메서드가 `JsonWebTokenError`을 반환합니다.
그렇기때문에 기존 조건문은 `true`가 됩니다.
기존 조건문이 성립되는 조건은 존재하지 않습니다. 
`refreshToken`은 에러 혹은 토큰 둘중하나이기때문에 
**`refreshToken`변수는 `falsy`가 절대 될 수 없습니다.**

변경한 조건문은 상단에서 두개의 초큰이 무조건 헤더에 포함되어있어야만하는 상황을 강제합니다.
때문에 `accessToken`의 만료 여부만 확인하면 됩니다.